recipes/xeus-sqlite/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "xeus-sqlite" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/jupyter-xeus/xeus-sqlite/archive/{{ version }}.tar.gz
+  sha256: eacc8591fd26c009a407d88867796880b13ea27113931157e4abe52324ba4b74
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
```

Comment:
```suggestion
    - cmake
    - make  # [unix]
    - nmake  # [win]
```

Reply:
Thanks for the review @marcelotrevisani :)